lib/usecases/uploadApplication.js

```diff
@@ -192,11 +192,10 @@ export default async ({
     }
   }
 
-  // const applicationStatus = await matchBusinessRatesNumber(
-  //   business.ratesAccountNumber
-  // ).then((value) => value);
-  // temporary fix until we can fix private key issues from env vars
-  const applicationStatus = 1;
+  const applicationStatus = await matchBusinessRatesNumber(
+    business.businessRatesAccountNumber
+  ).then((value) => value);
```

Comment:
This line seems strange to me, can't you just do
```suggestion
  );
```